cmd/internal/switcher/templates/test_data/dev/interfaces

```diff
@@ -75,29 +75,29 @@ iface vni104001
     vxlan-local-tunnelip 10.0.0.10
 
 # PXE-Config
-auto vlan4000
-iface vlan4000
+auto vlan966
+iface vlan966
     mtu 9000
     address 10.255.255.2/24
-    vlan-id 4000
+    vlan-id 966
     vlan-raw-device bridge
 
-auto vni104000
-iface vni104000
+auto vni10966
+iface vni10966
```

Comment:
The VNI for 966 should be vni100966 and not vni10966.

Reply:
So that it is always a six-digit VNI?